tests/utils/platform.ts

```diff
@@ -0,0 +1,29 @@
+import shell from 'shelljs';
+
+const isTestPlatformLinux = global.testPlatform === 'linux';
+const isTestPlatformMacOs = global.testPlatform === 'macos';
+const isTestPlatformWindows = global.testPlatform === 'windows';
+const isTestPlatformDocker = global.testPlatform === 'docker';
+const isTestPlatformEmpty = global.testPlatform == null;
```

Comment:
This should be using `globalThis`. And also, can you add a comment above that this is temporary until issue #435 is done. Afterwards these replaced as directories for grouping allowed tests.

Reply:
Use the docblock comment. Not `//`. Give it whitespace in between like:

```
/**
 *
 */

const isTestPlatformDocker ...;
const isTestPlatformMacOS ...;
const isTestPlatformWindows ...;

const isPlatformLinux ...;
```